snippets/logging-outputs-5.5-snip.adoc

```diff
@@ -0,0 +1,27 @@
+// Text snippet included in the following assemblies:
+//
+//
+// Text snippet included in the following modules:
+//
+//
+:_content-type: SNIPPET
+
+.Output Destinations
+[options="header"]
+|======
+|Feature|Protocol|Tested with|Fluentd|Vector
+|Cloudwatch|REST over HTTPS||✓|✓
+|Elasticearch||||
```

Comment:
This might require a fit of formatting on the table for ES versions 6,7 and 8 support. currently they appear as different entries on the table.